packages/shared/trade/src/trade-action-summary/trade-action-tool-tip.tsx

```diff
@@ -0,0 +1,34 @@
+import { Box, styled } from '@mui/material';
+import {
+  formatNumberAsPercent,
+  getDateString,
+} from '@notional-finance/helpers';
+import { H5 } from '@notional-finance/mui';
+import { TooltipProps } from 'recharts';
+
+const ToolTipBox = styled(Box)(
+  ({ theme }) => `
+  background: ${theme.palette.common.white};
+  padding: ${theme.spacing(2)};
+  text-align: left;
+  box-shadow: ${theme.shape.shadowStandard}
+`
+);
+
+export const TradeActionTooltip = (props: TooltipProps<number, string>) => {
```

Comment:
Are the chart tool tips necessarily custom like this or could it be re-usable and then we pass in the children content?